src/ngx_http_lua_bodyfilterby.c

```diff
@@ -218,13 +231,7 @@ ngx_http_lua_body_filter_file(ngx_http_request_t *r, ngx_chain_t *in)
     /*  make sure we have a valid code chunk */
     ngx_http_lua_assert(lua_isfunction(L, -1));
 
-    rc = ngx_http_lua_body_filter_by_chunk(L, r, in);
-
-    if (rc != NGX_OK) {
-        return NGX_ERROR;
```

Comment:
@agentzh Because `ngx_http_lua_body_filter_by_chunk` is return `NGX_OK` or `NGX_ERROR` only, so I think this `if` is unnecessary.
Also, this is ported from https://github.com/openresty/lua-nginx-module/commit/3f8a592a#diff-24e9ee6ce24592d8abdf568146073458L211